ray-operator/config/samples/ray-service.stable-diffusion.yaml

```diff
@@ -8,24 +8,23 @@ spec:
       - name: stable_diffusion
         import_path: stable_diffusion.stable_diffusion:entrypoint
         runtime_env:
-          working_dir: "https://github.com/ray-project/serve_config_examples/archive/d6acf9b99ef076a1848f506670e1290a11654ec2.zip"
-          pip: ["diffusers==0.12.1"]
```

Comment:
This version is not compatible with the dependencies in `rayproject/ray-ml:2.41.0.deprecated-gpu`.